cmd/clusterctl/clusterdeployer/bootstrap/kind/kind.go

```diff
@@ -24,10 +24,12 @@ import (
 
 	"github.com/pkg/errors"
 	"k8s.io/klog"
+
```

Comment:
This space should be removed

Reply:
From other opensource projecta, I think they follow this standard:
1. The standard library packages are always in the first group.
2. The third library packages are in second group
3. The project-specific packages are in third group

It seems `cluster-api` has different standard.

Reply:
We usually just adhere to `goimports` for simplicity, not a huge deal-breaker. But we should keep consistency with the other files in this project